docs/samples/python/customizing_presidio_analyzer.ipynb

```diff
@@ -581,6 +581,40 @@
     "The confidence score is now 0.4, instead of 0.01. We can change that by updating `zipcode_recognizer.CONTEXT_SIMILARITY_FACTOR` to something other than 0.4."
    ]
   },
+  {
```

Comment:
I think the entire example 6 needs some updates. It's now on the recognizer level and should be adapted to the analyzer level.

Reply:
@guybartal looks like the notebook has not been updated